client/consensus/nimbus-consensus/src/import_queue.rs

```diff
@@ -217,6 +218,7 @@ where
 	let block_import_for_queue = Box::new(NimbusBlockImport::new(
 		block_import,
 		with_delayed_best_block,
+		dont_create_gap,
```

Comment:
I would name this `create_gap` to be consistent with the parameter below. But it is just a personal preference, not important.

Reply:
I chose this because I like the default value of false to mean "new feature not enabled"